site/content/en/docs/Tasks/labels_and_annotations.md

```diff
@@ -2,7 +2,353 @@
 title: "Labels and Annotations"
 linkTitle: "Labels and Annotations"
 weight: 3
-date: 2017-01-05
+date: 2023-10-14
 description: >
   Working with Labels and Annotations
 ---
+
+A common set of Labels and Annotations can be applied to all Resources in a project by adding a `commonLabels` and a `commonAnnotations` entry to the `kustomization.yaml` file.
+
+## Add common Labels
```

Comment:
Something that I learned from reviewing this is that I think `commonLabels` can be deprecated, as it behaves the same as `labels.includeSelectors`. This doesn't change this documentation PR because we want to document `commonLabels` until it's actually deprecated.

@ncapps would you like to file an issue requesting the deprecation of `commonLabels` due to its overlapping behavior with `labels`? Of course, you'd have to agree with my sentiment and once you open the issue, others like @natasha41575 might have more context as to why we wouldn't want to deprecate. However, regardless of the outcome, I think this is a good learning opportunity? If not, I'm more than happy to file the issue.

Reply:
I created this issue https://github.com/kubernetes-sigs/kustomize/issues/5436 to track this duplicate behavior.

Reply:
Nice!